### ISSUE TITLE
TranslatableAdminExtension (PHPCR)

### ISSUE THREAD

**Issue by Infernosquad** [sonata-project/SonataTranslationBundle#48]
Just tried to set up translatable documents in my project with SonataAdminBundle and it didn't work until I uncommented following line in https://github.com/sonata-project/SonataTranslationBundle/blob/master/Admin/Extension/Phpcr/TranslatableAdminExtension.php#L43:
```php
 //@dbu : doesn't work - $documentManager->bindTranslation($object, $locale);
```
Why it doesn't work and commented out?

**Comment by nicolas-bastien** [comment]
Hi  @Infernosquad, as CMF is moving fast I won't be able to give a precise answer.

This line is the standard (at least documented) way to do it, but when I was working on it did.

We have projects in production working with this comment so I guess you are in an uptodate version.

You can make a PR if you can.

I'll check it in september cause I need to work test it seriously with other PR and tag new releases.

**Comment by nicolas-bastien** [comment]
Hi @Infernosquad , sorry for this so late reply !

Did you try to update your cmf bundle since last time ?
Does this still work ?

If yes you can make me a PR, I'll be happy to merge this. Thanks

**Comment by dbu** [comment]
fixed in #78
